stable/grafana/templates/deployment.yaml

```diff
@@ -43,8 +43,24 @@ spec:
 {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
 {{- end }}
-{{- if .Values.dashboards }}
+{{- if (.Values.persistence.enabled) or (.Values.dashboards) }}
       initContainers:
+{{- end }}
+{{- if .Values.persistence.enabled }}
+        - name: init-chown-data
+          image: busybox
+          imagePullPolicy: IfNotPresent
```

Comment:
The image location must have a tag (`1.30.0`) is the latest value. This is for reproducibility.

In addition, this needs to be templated. Look at `download-dashboards` for an example. there are places people want to run charts where they can't reach out to the internet or they have policies or pointing to corp container repos. This enables support for that.

Reply:
Sure thing, I'll add a tag and parameterize this in values.yaml.

Reply:
@mattfarina I pushed another commit which should resolve this, let me know if that looks good.